wcomponents-core/src/main/java/com/github/bordertech/wcomponents/layout/UIManager.java

```diff
@@ -62,15 +62,7 @@
 	/**
 	 * Marker Layout instance for when no layout was found.
 	 */
-	private static final Renderer NULL_RENDERER = new Renderer() {
-		/**
-		 * {@inheritDoc}
-		 */
-		@Override
-		public void render(final WComponent component, final RenderContext renderContext) {
-			// NO-OP
-		}
-	};
+	private static final Renderer NULL_RENDERER = (component, renderContext) -> { /* NO-OP */ };
```

Comment:
Not sure if we have a preference for explicitly providing param types or not so sure, why not.